packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -572,6 +653,43 @@ export const CommandMenu = () => {
                       </SelectableItem>
                     ))}
                   </CommandGroup>
+                  <CommandGroup heading="Tasks">
+                    {tasks?.map((task) => (
+                      <SelectableItem itemId={task.id} key={task.id}>
+                        <CommandMenuItem
+                          id={task.id}
+                          Icon={IconCheckbox}
+                          key={task.id}
+                          label={task.title ?? ''}
+                          onClick={() => openActivityRightDrawer(task.id)}
+                        />
+                      </SelectableItem>
+                    ))}
+                  </CommandGroup>
+                  {Object.entries(customObjectRecordsMap).map(
```

Comment:
I feel like we could simplify and reuse this loop for standard objects as well? (avoiding hardcoded company/task/etc)

Reply:
Ok, it still took 100lines to declare all objects as they have different Icon and label logics, but at least we arent repeating ourselves now !

Reply:
Better imho, thank you @ijreilly